Doc/library/os.path.rst

```diff
@@ -55,11 +55,14 @@ the :mod:`glob` module.)
 
 
 .. function:: abspath(path)
-
    Return a normalized absolutized version of the pathname *path*. On most
    platforms, this is equivalent to calling the function :func:`normpath` as
    follows: ``normpath(join(os.getcwd(), path))``.
-
+   
+   .. seealso:: 
+      :func:`join` - Used by abspath to combine paths
+      :func:`normpath` - Used by abspath to normalize paths
```

Comment:
Since we're already having a link to `normpath`, we may just have `

```rst
.. seealso:: :func:`os.path.join` and :func:`os.path.normpath`.
```

This should render as "See also os.path.join and os.path.normpath". If we mention normpath here, I think we can also simply write:

```rst
[...] is equivalent to calling ``normpath(join(os.getcwd(), path))``
```

Alternatively, we can say "See also [...]" at the text level and without using a directive.